src/fragments/lib/project-setup/ios/upgrade-guide/upgrade-guide.mdx

```diff
@@ -1,11 +1,9 @@
-As of v2, Amplify Library iOS is now Amplify Library for Swift. Amplify Library for Swift supports iOS 13+ and macOS 10.15+, and ships with APIs that leverage Swift Concurrency (async/await) to return values. There are a few things you need to know if you are upgrading to this latest version.
+As of v2, Amplify Library iOS is now Amplify Library for Swift. Amplify Library for Swift supports iOS 13+, macOS 10.15+, tvOS 13+, watchOS 9+, and visionOS (developer preview), and adopts Swift structured concurrency including async/await. There are a few things you need to know if you are upgrading to this latest version.
 
 ## What's changed
 
 We have re-written our APIs to support idiomatic Swift features including [structured concurrency](#structured-concurrency-asyncawait-pattern) and [combine support](/[platform]/start/project-setup/combine-framework/). We have also changed [how you access the underlying SDK](#escape-hatches) (keep in mind that the AWS SDK for Swift is still in Developer Preview). 
 
-The Predictions APIs do not currently have an upgrade plan. We will update this guide as soon as they do.
```

Comment:
Amplify Swift v2 was originally released without support for the Predictions category. That support has since been added.